Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM node:14.21.3-buster-slim
```

Comment:
Any specific reason to replace debian image with alpine?

Reply:
Just for smaller image size.

Reply:
let me fallback to buster-slim

Reply:
alpine image is notorious for its DNS issue ([ref](https://martinheinz.dev/blog/92)), lack of build toolchain ([ref](https://dev.to/kakisoft/dockeralpine-why-you-should-avoid-alpine-linux-44he)), and performance issue ([ref](https://medium.com/@kruchkov.alexandr/stop-using-alpine-images-b51d12b0fde2)). If this change doesn't aim to resolve any specific issue, I'd prefer to continue using slim Debian images.

Reply:
fixed. Thanks